pkg/interface/chat/src/js/api.js

```diff
@@ -181,6 +181,31 @@ class UrbitApi {
     });
   }
 
+  metadataAction(data) {
+    console.log(data)
+    return this.action("metadata-store", "metadata-action", data);
```

Comment:
Send this to metadata-hook. (My most recent PR notes that this is now the expected behavior)